.github/workflows/rebuild-nightly-docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Rebuild nightly docs
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
Does this capture `merged` PRs, but exclude those that are `closed` without being `merged`?

Reply:
That + https://github.com/FCP-INDI/C-PAC_tutorials/blob/2851af78eaaa21cb368bd5d990c522042b0855c0/.github/workflows/rebuild-nightly-docs.yml#L12 does